### PR TITLE
Changed Material Dialog with Alert Dialog for CoroutineHelpers.kt

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
## Purpose / Description
Replaces usages of MaterialDialog with AlertDialog in CoroutineHelpers.kt
## Fixes
Related to #13315 

## How Has This Been Tested?
Open each one of the changed dialogs on their respective preferences

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
LGTM - thanks!

File: AnkiDroid/src/main/java/com/ichi2/anki/CoroutineHelpers.kt [hunk 1]
```diff
@@ -21,17 +21,17 @@ import android.content.Context
 import android.view.WindowManager
 import android.view.WindowManager.BadTokenException
 import androidx.annotation.StringRes
+import androidx.appcompat.app.AlertDialog
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentActivity
 import androidx.lifecycle.coroutineScope
 import anki.collection.Progress
-import com.afollestad.materialdialogs.MaterialDialog
 import com.afollestad.materialdialogs.callbacks.onCancel
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/CoroutineHelpers.kt:29]: This also became unused.

File: AnkiDroid/src/main/java/com/ichi2/anki/CoroutineHelpers.kt [hunk 2]
```diff
@@ -356,11 +356,13 @@ suspend fun AnkiActivity.userAcceptsSchemaChange(): Boolean {
         return true
     }
     val hasAcceptedSchemaChange = suspendCoroutine { coroutine ->
-        MaterialDialog(this).show {
+        AlertDialog.Builder(
+            this
+        ).show {
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/CoroutineHelpers.kt:361]: Please keep the original formatting.

**Review by lukstbit — CHANGES_REQUESTED** [review]

**Review by lukstbit — APPROVED** [review]
Thank you!

**Pull request merged**